src/games/strategy/engine/random/ScriptedRandomSource.java

```diff
@@ -80,15 +80,6 @@ public int getRandom(final int max, final String annotation) throws IllegalState
     m_rolled += count;
     final int[] rVal = new int[count];
     for (int i = 0; i < count; i++) {
-      if (m_numbers[m_currentIndex] == PAUSE) {
```

Comment:
really the whole object should be a mock.. but otherwise this chunk can be removed for the moment.